packages/react-native-codegen/src/parsers/error-utils.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {ParserType} from './errors';
+
+const {
+  UnsupportedFunctionReturnTypeAnnotationParserError,
+} = require('./errors.js');
+
+function throwIfUnsupportedFunctionReturnTypeAnnotationParserError(
+  nativeModuleName: string,
+  flowReturnTypeAnnotation: $FlowFixMe,
```

Comment:
nit: can we change the name of the parameter with just `returnTypeAnnotation`?
At this point, we are not discriminating between the language in the params.

Thank you so much!

Reply:
Then should I remove the `returnTypeAnnotation` on line 25? Can both of them be handled from a common variable?

Reply:
I'm not sure I am following your suggestion. My comment was related to the name of the parameter we are using.
Can you expand on how will you remove the `returnTypeAnnotation`? Would you like to pass directly the `returnTypeAnnotation.type` value?

Reply:
I have made some changes to the parameter name. I hope that works. Please let me know what you think.